py-polars/tests/unit/datatypes/test_temporal.py

```diff
@@ -2166,6 +2166,15 @@ def test_tz_aware_with_timezone_directive(
     assert result == expected
 
 
+def test_local_time_zone_name() -> None:
+    ser = pl.Series(["2020-01-01 03:00ACST"]).str.strptime(
+        pl.Datetime, "%Y-%m-%d %H:%M%Z"
+    )
+    result = ser[0]
+    expected = datetime(2020, 1, 1, 3)
+    assert result == expected
```

Comment:
this already passed on `master`, but there was no test with `'%Z'` parsing a local time zone name